packages/firefox.rb

```diff
@@ -3,27 +3,28 @@
 class Firefox < Package
   description 'Mozilla Firefox (or simply Firefox) is a free and open-source web browser'
   homepage 'https://www.mozilla.org/en-US/firefox/'
-  version '81.0.1'
+  version '82.0'
   compatibility 'i686,x86_64'
   case ARCH
   when 'i686'
-    source_url 'https://archive.mozilla.org/pub/firefox/releases/81.0.1/linux-i686/en-US/firefox-81.0.1.tar.bz2'
-    source_sha256 'bcb562cb3a367c32710c2c19abaf63ad68846b9892795bf89592b879def917af'
+    source_url 'https://archive.mozilla.org/pub/firefox/releases/82.0/linux-i686/en-US/firefox-82.0.tar.bz2'
+    source_sha256 '2039a8e94fcda9bcad97d62059579d64744da522c7d65beac0edcbd2e128a06f'
   when 'x86_64'
-    source_url 'https://archive.mozilla.org/pub/firefox/releases/81.0.1/linux-x86_64/en-US/firefox-81.0.1.tar.bz2'
-    source_sha256 '7882ce6ece0b1aeb497d68e8aeff2c58eb5a926fb5003472db493c863e3129f9'
+    source_url 'https://archive.mozilla.org/pub/firefox/releases/82.0/linux-x86_64/en-US/firefox-82.0.tar.bz2'
+    source_sha256 '4305f43e72aa46ae42882eaa4e436b22c5142a4c9b2dc9f9ccc3292e18a5e39a'
   end
 
   case ARCH
   when 'i686', 'x86_64'
     depends_on 'gtk3'
     depends_on 'sommelier'
+    depends_on 'apulse'
```

Comment:
Can we put this dependency above sommelier?  If somebody doesn't have sommelier installed yet, we want them to see the postinstall instructions.

Reply:
Done !